Assets/Scripts/Speech Input Core Scripts/WindowsSpeechIn.cs

```diff
@@ -3,25 +3,30 @@
 using UnityEngine;
 using UnityEngine.Windows.Speech;
 
-public class WindowsSpeechIn : VoiceCommandBase {
```

Comment:
Could you undo reformatting related changes if possible? (only relavant commit should be highlited)

Reply:
alright, I reverted the other changes. Would you be alright with me making a seperate commit or PR using the default VS Code C# formatting?

Reply:
That would be ok. Just make sure you title the PR saying "[formatting] bla bla"